admission.bundle.yaml

```diff
@@ -75,7 +79,7 @@ spec:
       serviceAccountName: redis-enterprise-admission
       containers:
       - name: admin
-        image: redislabs/operator:6.0.6-11
+        image: redislabs/operator-internal:6.0.6-23
```

Comment:
```suggestion
        image: redislabs/operator:6.0.6-23
```

Reply:
I thought i fixed that, not sure how that happened thanks